src/suites/cts/validation/validation_test.ts

```diff
@@ -1,16 +1,103 @@
+import { unreachable } from '../../../framework/index.js';
+import { bindingTypeInfo } from '../format_info.js';
 import { GPUTest } from '../gpu_test.js';
 
+export enum BindingResourceType {
+  'uniform-buffer' = 'uniform-buffer',
```

Comment:
Maybe not worth it, but can this be `'uniform-buffer' = C.BindingType.UniformBuffer` ?'

Although it looks like we only use this as a type annotation and to do Object.keys(BindingResourceType).

Would it work to use the `C.BindingType` type annotation everywhere, and then have `export const BindingResourceTypes = [C.BindingType.UniformBuffer, ... ]` ?

Reply:
Good idea, will tighten the type from `string`.

Reply:
hm, actually this is kind of annoying: BindingResourceType is still not a subtype of GPUBindingType if you do that, so can't index `bindingTypeInfo` with it.